x-pack/plugin/searchable-snapshots/src/main/java/org/elasticsearch/index/store/cache/CacheFile.java

```diff
@@ -159,7 +169,7 @@ FileChannel getChannel() {
     }
 
     // Only used in tests
-    SortedSet<Tuple<Long, Long>> getCompletedRanges() {
+    public SortedSet<Tuple<Long, Long>> getCompletedRanges() {
```

Comment:
> Can we avoid making this public by adding a static method on `TestUtils` that can be used by tests residing outside the package?

Sure, I pushed 92d8ddacbbccb0ff1d78e8dab40120db0c52aa9d

> A sidenote question: it seems like there are some parts of searchable snapshots that are under `xpack.searchablesnapshots` and some that are not, any reason for that? Probably not the PR to refactor that though

The Lucene related classes have been originally located under the same package name (`org.elasticsearch.index.store`) as the one used in `core`and the plugin specific logic under the `xpack.pluginname` package like other plugins are doing. 

That's something we can revisit, specially since the plugin has grown a lot.

Reply:
> A sidenote question: it seems like there are some parts of searchable snapshots that are under `xpack.searchablesnapshots` and some that are not, any reason for that? Probably not the PR to refactor that though

Fixed in #70814